cfme/fixtures/targeted_refresh.py

```diff
@@ -0,0 +1,32 @@
+"""
+The idea of this fixture is to pass some entity ids to
+:py:class:`utils.targeted_refresh.TargetedRefresh` and check the evm.log
+whether all expected targeted refreshes
+were triggered.
+
+register_targeted_refresh accepts attributes for entity that is expected to trigger refresh
+"""
```

Comment:
This description literally tells nothing to me. I'd expect to get high-level overview of the thing first and then some usage examples to get me started with.